README.md

```diff
@@ -1,56 +1,69 @@
-# Arcaflow Plugin Fio
+# Fio Plugin for Arcaflow
 
-This plugin executes the fio workload software with a given input schema defined in a yaml file.
+This plugin executes the [fio](https://fio.readthedocs.io/en/latest/index.html) workload
+with a given input schema defined in a yaml file.
 
-## Image Building
+> [!NOTE]
+> Not all Fio parameters are currently implemented. Please review the [schema
+> documentation](#autogenerated-inputoutput-documentation-by-arcaflow-docsgen-below)
+> below.
 
-You can change this plugin's image version tag in
-`.github/workflows/carpenter.yaml` by editing the
-`IMAGE_TAG` variable, and pushing that change to the
-branch designated in that workflow.
+## Input structure
 
-## Run the thing
+The input structure (YAML or JSON) is a near-match to a standard fio job file. A list of `jobs` is provide, with each item requiring a `name`, which is equivalent to a fio job file section name, and a `params` object with key-value pairs that match job parameters from an fio job file.
 
```

Comment:
It would be nice if the line length were limited.

"provide" should be "provided".

Reply:
Thanks; I overlooked the line length on this one.